docs/lbl/lbl.rst

```diff
@@ -132,6 +135,8 @@ Be careful to be consistent and not to give partial or contradictory inputs. ::
             verbose=verbose,
         )
 
+**NOTE:** Warning / New:  wstep = 'auto' is optimized for performances while ensuring accuracy,
```

Comment:
Hello ! Good idea to mention if directly on the doc, but maybe directly in 'calculate-multiple-molecules-spectrum' is not the right place [(see the page here)](https://radis.readthedocs.io/en/latest/lbl/lbl.html#calculate-multiple-molecules-spectrum) 

What about a dedicated section after [Advanced/Computation Parameters](https://radis.readthedocs.io/en/latest/lbl/lbl.html#computation-parameters), which could be "Choose the right wavenumber grid" explaining there is a tradeoff in between accuracy and performance. 

Have a look at the small edits I made in your function docstrings : they show how to add a 'note' , and also how to refer to the warning parameters, for instance. 

Side note : if you edit lbl.rst, on the same files some links are broken, for instance 
```
:py:func:`~radis.lbl.calc.eq_spectrum_gpu`
``` 
which should be 
```
:py:meth:`~radis.lbl.factory.SpectrumFactory.eq_spectrum_gpu`
```

Reply:
Added the new section `Choose the right wavenumber grid` as you mentioned. You may want to look into its content and modify it accordingly.

Reply:
It's good and clear. I just added a commit suggestion